File: app/src/main/java/omniex/nl/omniex/ui/app/product/list/ProductsListView.java
package omniex.nl.omniex.ui.app.product.list;

import java.util.List;

import nl.omniex.omniexshopping.data.model.products.Product;
import nl.omniex.omniexshopping.ui.base.BaseView;

public interface ProductsListView extends BaseView {
    void onProductsListFetched(List<Product> productList);
}

File: app/src/main/java/omniex/nl/omniex/ui/app/product/list/ProductsListView.kt
package omniex.nl.omniex.ui.app.product.list

import omniex.nl.omniex.data.model.products.Product
import omniex.nl.omniex.ui.base.BaseView

interface ProductsListView : BaseView {
    fun onProductsListFetched(productList: List<Product>)
}
